lib/ui/painting/flutter_rtree.cc

```diff
@@ -0,0 +1,243 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/painting/flutter_rtree.h"
+#include "flutter/fml/logging.h"
+
+namespace flutter {
+
+FlutterRTree::FlutterRTree() : fCount(0) {}
```

Comment:
Please follow the engine style guide for new code. I take it that we will now be owning this moving forward.